admin_guide/cluster_management.rst

```diff
@@ -487,23 +493,85 @@ To set non-sensitive libpq connection parameters to be used for all node connect
 
   ALTER DATABASE foo
```

Comment:
What was the reason we have `ALTER DATABASE foo` here. During testing I always used `ALTER SYSTEM`. Does it make sense to change that here?

Reply:
This was already in here, @begriffs is best to answer the question on historical reasons I guess.

Reply:
There's no deep history -- I introduced it in 626fab09. If alter system is preferred then it's fine to use that.

Reply:
Okay, pushed commit that changes it to `ALTER SYSTEM`. That way we're sure it works.